src/documents/LocalizationInThePreferencesFramework.md

```diff
@@ -232,3 +232,29 @@ The strings can also be accessed directly, outside of the context of IoC referen
 ```javascript
 that.msgLookup.lookup(value); // where value is either the string name or the key in the stringArrayIndex to lookup.
```

Comment:
But it seems we don't quite use it in the code. Instead, the IoC references like ```{that}.msgLookup.slidingPanelShowText``` are used.